fhir-bucket/src/main/java/com/ibm/fhir/bucket/client/FhirClientUtil.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.bucket.client;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+
+import com.ibm.fhir.model.format.Format;
+import com.ibm.fhir.model.generator.FHIRGenerator;
+import com.ibm.fhir.model.generator.exception.FHIRGeneratorException;
+import com.ibm.fhir.model.resource.Resource;
+
+/**
+ * Utilities for working with the FHIR client
+ */
+public class FhirClientUtil {
```

Comment:
Per convention let's capitilize 
Also FHIRBulkClientUtil (scoping it in typeaheads)

Reply:
done